### ISSUE TITLE
${__cell} doesn't work anymore in the new table panel in grafana 7.0

### ISSUE THREAD

**Issue by fredduwez** [grafana/grafana#25065]
**What happened**:
${__cell} doesn't work anymore in the new table panel in grafana 7.0
So I can't migrate my old tables, nor create new ones as the old table is deprecated.

**What you expected to happen**:
When I set a data link for a column, now by using the Override menu of the table panel, I expect the value of the cell to replace the variable ${__cell}, so I can link the row to another dashboard.

**Environment**:
- Grafana version: 7.0 hosted by granafa.net

**Comment by torkelo** [comment]
use ${__field.raw} or ${__field.text} ${__field.numeric}   (why so many options, depends on if you have value mappings, display formatting like units etc)

**Comment by fredduwez** [comment]
@torkelo: thx but I still can't make it work; is there a documentation about this?

**Comment by sabeDev** [comment]
This is possible, I am using 9.1.7 and originally used the old table; however, after migrating noticed the same issue. That said, I found a way to make this work:

Add the ID column that you want to hide; example simID
If the column is the first column, say 0; then use: var-whatever=${__data.fields[0]}
Now, override the column, field with name: simID
Add option ‘Hide in Table’ and turn on
I also had to format the number, e.g. remove decimals & set unit to standard
Hope that helps someone… took me a good 2 hours to figure.

Btw, it also works on graphs and you do not have to hide; aka reference your original query in the datasource and add a data link with the same options.
